templates/concourse-worker.service.j2

```diff
@@ -4,12 +4,14 @@
 Description=concourse-worker
 Requires=network-online.target
 After=network-online.target
+Wants=concourse-worker-watchdog.service
```

Comment:
Maybe we could add a if condition in this template, as the watchdog is not always enabled eg https://github.com/cycloid-community-catalog/stack-external-worker/blob/master/ansible/external-worker.yml#L202-L215

Reply:
True, forgot about that.

Reply:
done